### ISSUE TITLE
Minor Scaling Problem with the default zoom value?

### ISSUE THREAD

**Issue by hikarin-dev** [ngld/OverlayPlugin#152]
![image](https://user-images.githubusercontent.com/19721540/89724918-be5e5400-da3b-11ea-9d23-e19201750f7f.png)

The scale of the render doesn't seem to be at exactly 100%, I tried the default zoom value and adjusting it by pixel and I couldn't match the image on the left (from ACTWebSocket). Its slightly more noticeable in the header grid pattern as well.

**Comment by ngld** [comment]
Looks fine on my machine. This is the preview I get when I add a new MopiMopi overlay:
![Advanced_Combat_Tracker_AfN03xrfFf](https://user-images.githubusercontent.com/5465775/89738879-59771e00-da7c-11ea-9d2c-b32ed11e93e4.png)

Did you try resetting the zoom?

Just in case I'm misunderstanding your problem: The missing space on the right is caused by reducing the zoom below 100%. The default zoom of 100% matches what a browser (i.e. Chrome) displays for https://haeruhaeru.github.io/mopimopi/ so I'd consider it correct. If ACTWebSocket renders overlays smaller than any other browser, I'd say that's a bug on ACTWebSocket's part.

**Comment by hikarin-dev** [comment]
Yes, I tested with the default zoom and the zoom reset, like I wrote, it is just very minor, and slightly more clear if the background is a grid, the image I showed is using a different resolution in the overlay's own settings so y-axis have gotten affected too, but here are 2 more images at the default resolution of the overlay's own settings. First is from ngld OverlayPlugin, second from Chrome, and third from Firefox. I filled the grid w/ higher contrast color so its more visible.
![image](https://user-images.githubusercontent.com/19721540/89742800-a9330500-dacf-11ea-8fb2-99d257b0f180.png)
I apologize if its a non-issue, it's just my ocd that can't unsee the detail

**Comment by hibiyasleep** [comment]
I've reproduced this issue, seems related to ACT's window size - if it's (any odd number)px, scale slider doesn't stay at 1.000 point, so causing this. reproduced at 1045px, coudln't at 1044px.

![image](https://user-images.githubusercontent.com/5380174/92402544-b264cf80-f16a-11ea-9e16-a9841d16b9ab.png)

**Comment by ngld** [comment]
@gynryiie Can you check if 0.16.5 fixes your issue?
